arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -361,41 +361,48 @@
       "required": [ "Name" ],
       "properties": {
         "Name": {
-          "description": "Name of the offer term. For example: 'Monetary Credit', 'Monetary Commitment",
-          "type": "string"
+          "description": "Name of the offer term",
+          "type": "string",
+          "enum": [
+            "Recurring Charge",
+            "Monetary Commitment",
+            "Monetary Credit"
+          ]
         },
         "EffectiveDate": {
-          "description": "Indicates the date from which the meter rate or offer term is effective.",
+          "description": "Indicates the date from which the offer term is effective.",
           "type": "string",
           "format": "date-time"
-        },
-        "ExcludedMeterIds": {
-          "description": "An array of meter ids that are excluded from the given offer terms.",
-          "type": "array",
-          "items": {
-            "type": "string",
-            "format": "uuid"
-          }
         }
       }
     },
     "MonetaryCredit": {
       "x-ms-discriminator-value": "Monetary Credit",
+      "description": "Indicates that a that this is a monetary credit offer.",
       "allOf": [
         {
           "$ref": "#/definitions/OfferTermInfo"
         }
       ],
       "properties": {
         "Credit": {
-          "description": "The amount of credit provided under the terms of the given offer level. This field is used only by offer terms of type 'MonetaryCredit'.",
+          "description": "The amount of credit provided under the terms of the given offer level.",
           "type": "number",
           "format": "decimal"
+        },
+        "ExcludedMeterIds": {
+          "description": "An array of meter ids that are excluded from the given offer terms.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "uuid"
```

Comment:
This is a list of Meter ids for which the monetary credit can not be applied against, these will be correlated with the MeterId in the MeterInfo object which is also a uuid.